components/automate-ui/src/app/entities/userperms/userperms.effects.ts

```diff
@@ -47,4 +50,10 @@ export class UserPermEffects {
     mergeMap((action: GetUserParamPerms) => this.requests.fetchParameterized(action.payload).pipe(
       map((resp: UserPermsResponsePayload) => new GetUserParamPermsSuccess(resp.endpoints)),
       catchError((error: HttpErrorResponse) => of(new GetUserParamPermsFailure(error))))));
+
+  @Effect()
+  fetchApplyRulesStatusAllowed$ = interval(1000 * POLLING_INTERVAL_IN_SECONDS).pipe(
+    mergeMap(() => this.requests.fetchSome({ paths: [ProjectConstants.APPLY_RULES_ENDPOINT] }).pipe(
+      map((resp: UserPermsResponsePayload) => new GetSomeUserPermsSuccess(resp.endpoints)),
```

Comment:
If the other PR https://github.com/chef/automate/pull/1162 is ensuring all users have access to the apply-rules status endpoint, this extra introspection call seems unnecessary.